qpc/utils.py

```diff
@@ -16,6 +16,8 @@
 import os
 import json
 from xdg.BaseDirectory import xdg_data_home, xdg_config_home
+from qpc.translation import _ as t
```

Comment:
Why import `_` as `t` when it is `_` in all of the other modules?

Reply:
there was an existing use of `_` within the file, so the linter called out a shadow variable issue. Seemed simple enough to just do the above since it was such a limited usage.